lib/cocoapods/target/build_settings.rb

```diff
@@ -585,13 +588,20 @@ def initialize(target, non_library_spec = nil, configuration: nil)
         define_build_settings_method :static_frameworks_to_import, :memoized => true do
           static_frameworks_to_import = []
           static_frameworks_to_import.concat vendored_static_frameworks.map { |f| File.basename(f, '.framework') } unless target.should_build? && target.build_as_dynamic?
+          static_frameworks_to_import.concat vendored_xcframeworks
+                                                .select { |f| f.slices.any? { |slice| !Xcode::LinkageAnalyzer.dynamic_binary?(slice.binary_path) }}
```

Comment:
instead of `any?` can we use `none?`

Reply:
yes! honestly didn't realize `none?` was a thing

Reply:
ah nevermind, `none?` wouldn't many sense here because we need _at least_ one slice for which `dynamic_binary?` is false. we don't need them _all_ to be `false`